src/ripple/protocol/STObject.h

```diff
@@ -970,6 +971,27 @@ STObject::operator[](OptionaledField<T> const& of) ->
     return OptionalProxy<T>(this, of.f);
 }
 
+inline
+NetClock::time_point
+getsfCloseTime(STObject const& sle)
```

Comment:
`(*sle)[sfCloseTime]` should return `NetClock::time_point`, there's an established mechanism for making this work.